index-handlers/src/test/java/no/sikt/nva/nvi/index/aws/OpenSearchClientTest.java

```diff
@@ -575,7 +577,25 @@ void organizationAggregationShouldNotContainAggregationsForOtherTopLevelOrgs()
         assertThat(orgIds, not(containsInAnyOrder(NTNU_INSTITUTION_ID.toString())));
     }
 
-    @NotNull
+    @Test
+    void shouldExcludeFields() throws IOException, InterruptedException {
+        var document = documentWithContributors();
+        addDocumentsToIndex(document);
+        var searchParameters = defaultSearchParameters()
+                                   .withExcludeFields(List.of("publicationDetails.contributors"))
+                                   .build();
+        var searchResponse = openSearchClient.search(searchParameters);
+        var firstHit = searchResponse.hits().hits().get(0).source();
+        assertNull(firstHit.publicationDetails().contributors());
```

Comment:
This produces a linter warning because `publicationDetails` could be null. Not actually a problem, but perhaps add an assertion or wrapper to suppress the warning? There are a few similar cases in the same file too:

```
- Method invocation 'createdDate' may produce 'NullPointerException' :184
- Method invocation 'createdDate' may produce 'NullPointerException' :185
- Method invocation 'approvals' may produce 'NullPointerException': 360
- Method invocation 'publicationDetails' may produce 'NullPointerException': 582
```